api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/activation/ELU.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2021 JetBrains s.r.o. and Kotlin Deep Learning project contributors. All Rights Reserved.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlinx.dl.api.core.layer.activation
+
+import org.tensorflow.Operand
+import org.tensorflow.op.Ops
+
+/**
+ * Exponential Unit activation function.
+ *
+ * It follows:
+ * ```
+ * f(x) = x,                    if x > 0
+ * f(x) = alpha * (exp(x) - 1), if x <= 0
+ * ```
+ *
+ * In contrast to ReLU it has negative values which push the mean of
+ * the activation closer to zero which enable faster learning as they
+ * bring the gradient to the natural gradient.
+ *
+ * @property [alpha] Hyperparameter that controls the value to which
+ * an ELU saturates for negative net inputs. Should be > 0.
+ * @constructor Creates [ELU] object.
+ * @since 0.3
+ */
+public class ELU(
+    public val alpha: Float = 1.0f,
```

Comment:
looks like this parameter is the reason for the separate ELU layer in Keras (like for ReLU too) because the simplest activation usually have no parameters

Reply:
that convinces me that it could be somehow "duplicated code" but in fact with other features and use cases